packages/costa/src/runnable.ts

```diff
@@ -155,12 +169,36 @@ export class PluginRunnable {
       path.join(installDir, 'node_modules', pkg.name),
       compPath + `/node_modules/${pkg.name}`);
 
-    const resp = await this.sendAndWait(PluginOperator.WRITE, {
-      event: 'start',
-      params: [
-        pkg,
-        ...args
-      ]
+    // wrap the PNR with the start logic.
+    const resp = await new Promise<PluginMessage>(async (resolve, reject) => {
```

Comment:
I think using `await` in `Promise` is not a good practice.

Reply:
Is there a better way this?